src/initialoutputs/states.txt

```diff
@@ -11,8 +11,8 @@ Main Tree Parent Node,""
 Trade,Who do you want to tradeUseCase with?
 Pick Player (Trade),What property do you want from the player?
 Trader/Tradee Has No Properties,Trade cannot be done; one of you have no properties
```

Comment:
Not a big deal but capitalization is a little inconsistent here

Reply:
The capitalization is inconsistent with the key of the hashmap, and if I change the key value, its going to be very tedious to trace through the entire program to find where this use case is use.